tests/env.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+"""
+Tests for the Chatshare application.
+"""
+
+import unittest
+import os
+
+class TestEnvironmentVariablesLoaded(unittest.TestCase):
```

Comment:
Do we need this? It's testing the external .env loader dependency which should have its own tests.

Reply:
The issue is more about the fact that we now cannot change the name or content of that environment variable without breaking a test which doesn't add much additional confidence, but increases the brittleness of testing.